Gems/ScriptAutomation/Code/Source/ScriptAutomationScriptBindings.cpp

```diff
@@ -20,34 +20,40 @@ namespace ScriptAutomation
 {
     namespace Bindings
     {
-        void Print(const AZStd::string& message)
+        void Print(const AZStd::string& message [[maybe_unused]])
         {
+#ifndef RELEASE //AZ_TracePrintf does nothing in release, ignore this call
```

Comment:
The define is `_RELEASE`

Reply:
It seems both `RELEASE` and `_RELEASE` are in use
`RELEASE`
https://github.com/o3de/o3de/blob/eb231eaa0b1ed359f1bd262fc36999c613b9683a/Code/Framework/AzCore/AzCore/Debug/Trace.h#L284
`_RELEASE`
https://github.com/o3de/o3de/blob/eb231eaa0b1ed359f1bd262fc36999c613b9683a/Code/Framework/AzCore/AzCore/Debug/Budget.h#L52

Reply:
Underscore release is a lot more common than release.
It seems like we add
[RELEASE](https://sourcegraph.com/search?q=context:global+repo:github%5C.com/o3de/o3de+file:.*%5C.cpp%24+%5CbRELEASE%5Cb&patternType=regexp&case=yes)
[_RELEASE](https://sourcegraph.com/search?q=context:global+repo:github%5C.com/o3de/o3de+file:.*%5C.cpp%24+%5Cb_RELEASE%5Cb&patternType=regexp&case=yes)